package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run start:prod || npm run start:dev",
-    "start:dev": "webpack-dev-server --inline --content-base public/ --history-api-fallback",
-    "start:prod": "webpack && node server.js"
+    "start:dev": "webpack-dev-server --inline --hot --content-base public/ --history-api-fallback",
```

Comment:
It enables hot-reloading. If you write a component, it will automatically refresh and implement the changes.